src/filter.rs

```diff
@@ -134,12 +126,18 @@ fn set_bit(bit: usize, buf: &mut [u8]) {
     buf[byte] |= 1 << bit_in_byte;
 }
 
-fn optimal_num_probes(bits_per_key: u32) -> u16 {
+pub(crate) fn optimal_num_probes(bits_per_key: u32) -> u16 {
```

Comment:
```suggestion
fn optimal_num_probes(bits_per_key: u32) -> u16 {
```
I don't think this is necessary?

Reply:
If we don't extract this part into a separate function here, we would have to manually reimplement the hashfilter size calculation again. I think it would be better to extract it into a separate function.

Reply:
agreed for `filter_size_bytes`, but I don't see where `optimal_num_probes` is used outside of this?